### PR TITLE
docs: open codesandbox always use jsx

### PR DESCRIPTION
**Pull request by xiaohuoni**
Open codesandbox always use jsx

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ const eslintrc = {
     'react/static-property-placement': 0,
     'jest/no-test-callback': 0,
     'jest/expect-expect': 0,
+    'import/extensions': 0,
```
Comment (afc163) [inline .eslintrc.js]: 别禁用这个。

Comment (zombieJ) [reply .eslintrc.js]: 这个 rule 对我们来说没什么用，一堆 rc 库是 js 的。test case 又是 js 引用 tsx，改了也没什么明显的好处。我建议禁用掉。

Comment (xiaohuoni) [reply .eslintrc.js]: 你们看一下要怎么改？

**Comment by zombieJ** [comment]
Seems not work: https://deploy-preview-20154--ant-design.netlify.com/components/table-cn/#components-table-demo-virtual-list

**Pull request merged**